style.css

```diff
@@ -183,6 +183,54 @@ table {
   min-height: 5vh;
 }
 
+/* menu strt */
+.m-menu {
```

Comment:
- [ ] Good job so far 👍 your menu works as expected but I have a small issue with the design 😢  Please make sure to that your mobile menu design is similar to the figma template. Your **Xmark** and menu option spacing are a bit diffrent, please check the screenshots below ⬇️  to see the differences and fix it 👍 

#### Figma design
![FIGMA](https://user-images.githubusercontent.com/6615394/223278037-29cec3e6-5800-422d-88fe-5fb6ace61b32.jpg)

#### Your design
![MENU](https://user-images.githubusercontent.com/6615394/223278071-972f30e8-dc09-4178-988a-064f1cdd8ed9.jpg)